src/custom/constants/GPv2Settlement.json

```diff
@@ -47,18 +47,5 @@
     ],
     "name": "Trade",
     "type": "event"
-  },
-  {
-    "inputs": [],
-    "name": "allowanceManager",
```

Comment:
There is also a `.vaultRelayer()` call for getting the vault relayer in an analogous way.

Reply:
yep, I killed that part of the ABI cause we don't need it. We get the address from the NPM package.